lenet/index.html

```diff
@@ -0,0 +1,118 @@
+      <div class="row" style="margin-top:20px;">
+        <div class="col">
+        </div>
+        <div class="col-4" align="center">
+          <canvas id="visual_canvas" width=280 height=280></canvas>
+        </div>
+        <div class="col-4" align="center">
+          <div class='row text-center tc' id='inferencetitle'>
+            <h5>Prediction Result:</h5>
+          </div>
+          <div class='row text-center' id='inferenceresult'>
+            <table class='table table-sm'>
+              <thead>
+                <tr>
+                  <th scope='col'>#</th>
+                  <th scope='col'>Label</th>
+                  <th scope='col'>Probability</th>
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <th scope='row'>1</th>
+                  <td id='label0'></td>
+                  <td id='prob0'></td>
+                </tr>
+                <tr>
+                  <th scope='row'>2</th>
+                  <td id='label1'></td>
+                  <td id='prob1'></td>
+                </tr>
+                <tr>
+                  <th scope='row'>3</th>
+                  <td id='label2'></td>
+                  <td id='prob2'></td>
+                </tr>
+              </tbody>
+            </table>
+          </div>
+          <div class='row text-center tc' id='inferencetitle'>
+            <div id='compilationTime'></div>
+          </div>
+          <div class='row text-center tc' id='inferencetitle'>
+            <div id='inferenceTime'></div>
+          </div>
+        </div>
+        <div class="col">
+        </div>
+      </div>
+      <div class="row" style="margin-top:50px;">
+        <div class="col">
+        </div>
+        <div class="col-1">
+          <button type="button" class="btn btn-primary btn-lg" id="next">Next</button>
+        </div>
+        <div class="col-1">
+          <button type="button" class="btn btn-primary btn-lg" id="clear">Clear</button>
+        </div>
+        <div class="col-1">
+          <button type="button" class="btn btn-primary btn-lg" id="predict" disabled>Predict</button>
+        </div>
+        <div class="col-4"></div>
+        <div class="col">
+        </div>
+      </div>
+    </div>
+    <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
+    <script src="https://webmachinelearning.github.io/webnn-polyfill/dist/webnn-polyfill.js"></script>
```

Comment:
are you planning to publish the webnn plyfill package as an npm?

Reply:
It sounds good. Once the polyfill merged, I could that.

Reply:
file issue https://github.com/webmachinelearning/webnn-polyfill/issues/3 to track it.